spec/forms/collection_edit_form_spec.rb

```diff
@@ -67,4 +68,23 @@
       end
     end
   end
+
+  context 'while logged in' do
+    let(:user_attributes) do
+      { email: 'test@example.com' }
+    end
+
+    let(:user) do
+      User.new(user_attributes) { |u| u.save(validate: false) }
+    end
+
+    before do
+      login_as user
+    end
+
+    it 'visibly marks the title field as required' do
+      visit '/concern/generic_works/new'
+      expect(page).to have_content('Title required')
```

Comment:
This sort of thing should be done in a feature test `spec/features/*`, not in a unit test.